seatunnel-e2e/seatunnel-spark-connector-v2-e2e/src/test/java/org/apache/seatunnel/e2e/spark/v2/iotdb/FakeSourceToIoTDBIT.java

```diff
@@ -17,7 +17,7 @@
 
 package org.apache.seatunnel.e2e.spark.v2.iotdb;
 
-import static org.awaitility.Awaitility.given;
+import static org.testcontainers.shaded.org.awaitility.Awaitility.given;
```

Comment:
Where is the problem?

Reply:
Need to error details

Reply:
![image](https://user-images.githubusercontent.com/51053924/187625065-33cce14f-b068-4cf4-9a7b-069aae8e893f.png)

It seems that I can not download this version dependency. I will try to exchange a another maven repo again.

BTW, I see this test case use `org.testcontainers.shaded.org.awaitility.Awaitility.given`, should we unified this?

![image](https://user-images.githubusercontent.com/51053924/187625674-72f5a192-1e7f-4253-ab12-3888b6a43ade.png)

Reply:
![image](https://user-images.githubusercontent.com/51053924/187645940-d509950e-c78f-45fc-add5-14ad02c52b46.png)

![image](https://user-images.githubusercontent.com/51053924/187646154-c6f3072f-9117-4910-b0d4-b27d016ad62d.png)


Change maven repo to default it also has a error...

Reply:
Could you try to download this manually, on my local it is normal.

Reply:
This may be the problem of my computer, I changed a computer this problem solved. I revert these files.